cogboard-webapp/src/components/widgets/index.js

```diff
@@ -200,6 +202,14 @@ const widgetTypes = {
     component: LinkListWidget,
     dialogFields: ['LinkListItems'],
     initialStatus: 'NONE'
+  },
+  PullRequestReminderWidget: {
+    component: PullRequestReminderWidget,
+    dialogFields: ['EndpointField', 'SchedulePeriod', 'Path'],
```

Comment:
Let's expand dialog with new fields:
1 `Project (Group)` 2 `Repository` in place of `Path`
3 `API Type` dropdown with values: `Bitbucket 1.0`, `GitHub` - we will expand if needed

Reply:
Use `SonarQube Version` field as an example